Gemfile

```diff
@@ -5,14 +5,15 @@ source 'https://rubygems.org'
 gemspec name: 'mongoid-slug'
 
 case (version = ENV['MONGOID_VERSION'] || '8')
-when 'HEAD'
+when /\Ahead\z/i
   gem 'mongoid', github: 'mongodb/mongoid'
 when /\A\d+\z/
   gem 'mongoid', "~> #{version}.0"
 else
   gem 'mongoid', version
 end
 
+gem 'mongoid_paranoia', '>= 0.6'
```

Comment:
You probably want to make a separate integration test that uses a different Gemfile for mongoid_paranoia to make sure it works without.

Reply:
ok good point. same thought crossed my mind.